libraries/Microsoft.Bot.Builder.AI.QnA/AnswerSpanRequest.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    public class AnswerSpanRequest
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether gets or sets the enablet.
```

Comment:
> /// Gets or sets a value indicating whether gets or sets the enablet. [](start = 7, length = 70)

This summary is not clear. Please improve.

Reply:
Updated as per comment.

---
In reply to: [433508388](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r433508388) [](ancestors = 433508388)